server/examples/CreateOrder.http

```diff
@@ -0,0 +1,9 @@
+POST http://localhost:3001/api/orders HTTP/1.1
+Content-Type: application/json
+
+{
+    "clientId" : "318d971d89d6240eb03742d7",
+    "products" : [
+    { "productId": "718d971d89d6240eb03742d7", "quantity": 3 },
```

Comment:
This input payload is wrong. The right format is:
{  
  "clientID": "....",
  "products": {
          "718d971d89d6240eb03742d7":3,
          "298d971d89d6240eb03742d7":4,
          }
}